Header header
std_msgs/Int32 id
State[] states
std_msgs/String uid